src/Sylius/Bundle/AdminBundle/Command/CreateAdminUserCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Command;
+
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\Locale\Model\Locale;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'sylius:admin-user:create',
+    description: 'Create a new admin user'
+)]
+class CreateAdminUserCommand extends Command
+{
+    private SymfonyStyle $io;
+
+    public function __construct(
+        private UserRepositoryInterface $adminUserRepository,
+        private RepositoryInterface $localeRepository,
+        private FactoryInterface $adminUserFactory,
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->io->title('Admin user creation');
+
+        $confirm = $this->io->confirm('Do you want to create an admin user ?', true);
+
+        if (!$confirm) {
+            return Command::INVALID;
+        }
+
+        do {
+            $email = $this->io->ask('Email', null);
+        }
+        while (!$this->validateEmail($email));
```

Comment:
Maybe create a Question and use `->setValidator` and `setMaxAttempts` instead of looping indefinetely?
https://symfony.com/doc/current/components/console/helpers/questionhelper.html#validating-the-answer